### ISSUE TITLE
KC-07223 and T4-2634

### ISSUE THREAD

**Issue by eisoch** [eisoch/irg#65]
KC-07223 and T4-2634 are both ⿱山田. 

If TCA disunified T4-2634 from U+5CC0 and moved it to a new code point, ROK could do the HE.

**Comment by kenlunde** [comment]
Was KC-07223 removed from IRG Working Set 2017?

**Comment by hfhchan** [comment]
I don't think so.  I can't find it in 2015 or 2017 (at least not in the online version).

But KC-07223 can be found here:
http://www.koreanhistory.or.kr/newchar/grid_list.jsp?code_type=3&codebase=KC07223

**Comment by hfhchan** [comment]
For completeness, code chart of U+5CC0 as follows.
![image](https://user-images.githubusercontent.com/8191296/62409509-30bb7d80-b60b-11e9-96b8-cb2ff93657fe.png)
